### PR TITLE
Sam/README Updates

### PR DESCRIPTION
**Pull request by SPerrott22**
moved interns to current contributors and moved people no longer on dev team to past contributors.

Fixes issue #380

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,20 +15,29 @@ There is also hardware available for hackers to check out and hack with.
...
 - Jenna Wang [(@ariyin)](https://github.com/ariyin)
+- James Wu [(@jamesmwu)](https://github.com/jamesmwu)
 - Shiyu Ye [(@Monicaaawa)](https://github.com/Monicaaawa)
-- Brooke Jiang [(@jiangxuann)](https://github.com/jiangxuann)
-- Jonathan Si [(@jsi19)](https://github.com/jsi19)
-- Andy Lewis [(@datowq)](https://github.com/datowq)
-- Satyen Subramaniam [(@SubramaniamSatyen)](https://github.com/SubramaniamSatyen)
 
 Past contributors: 
+- Satyen Subramaniam [(@SubramaniamSatyen)](https://github.com/SubramaniamSatyen)
```
Comment (katelynsyu) [inline README.md:35]: nit: can we order this in alphabetical order by first name?

**Review by katelynsyu — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -15,20 +15,29 @@ There is also hardware available for hackers to check out and hack with.
 This website is developed with [Gatsby](https://www.gatsbyjs.org/) and
 [Material UI](https://material-ui.com/), and is a work-in-progress being
 developed by ACM Hack’s Dev Team: 
-- Alex Xia [(@khxia)](https://github.com/khxia)
+- Aazel Tan [(@aazeltan)](https://github.com/aazeltan)
```
Comment (nathanzzhang) [inline README.md:18]: Nice work! Typically, we have the current dev team leads (Jakob, Katelyn, Maggie) at the top of the list. Members thereafter are typically alphabetized, too! Do you mind changing the order of the contributors, thanks Sam!

**Review by nathanzzhang — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (SPerrott22) [reply README.md]: Done. See commit 46f76f2

File: README.md (hunk 2 above)
Comment (SPerrott22) [reply README.md:21]: Yes. See commit 46f76f2

**Review by nathanzzhang — APPROVED** [review]
lgtm!

**Pull request merged**